samples/bluetooth/peripheral_ots/sample.yaml

```diff
@@ -7,6 +7,7 @@ tests:
     platform_allow:
       - qemu_cortex_m3
       - qemu_x86
+      - rd_rw612_bga
```

Comment:
I know it was mentioned earlier that you need more coverage, but I don't think it's right to add the board to absolutely every single sample application. We're not doing that for most HCI drivers, so I don't think we should do it for this new one either. You *should* have some coverage though, ideally so that you cover all the possible build configurations for the driver. These samples will not get exercised at runtime by upstream CI for this board, so the key thing is really the build time coverage - enabling new samples which don't increase the coverage is not helpful (just increases CI completion time without any benefit)

Reply:
Most of the samples weren't built by the CI since they don't belong to the "integration platforms" list. Your comment still makes sense, I have removed the platform name from all the samples/tests except peripheral_ht, central_ht and tester application.
Thanks a lot.